server/sonar-ce/src/main/java/org/sonar/ce/container/ComputeEngineContainerImpl.java

```diff
@@ -646,6 +199,473 @@ protected ComponentContainer getComponentContainer() {
     return level4;
   }
 
+  private static Object[] level1Components() {
+    Version apiVersion = ApiVersion.load(System2.INSTANCE);
+    return new Object[] {
+      ComputeEngineSettings.class,
+      new SonarQubeVersion(apiVersion),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "SonarQubeVersion"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)